apps/emqx/src/emqx_ds_schema.erl

```diff
@@ -246,7 +246,7 @@ fields(layout_builtin_reference) ->
                 reference,
                 #{
                     'readOnly' => true,
-                    importance => ?IMPORTANCE_HIDDEN
+                    importance => ?IMPORTANCE_LOW
```

Comment:
Having reference layout in the release build is useful. However hocon complains that a non-hidden record cannot have all of its fields hidden.